packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1931,7 +1931,7 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
                     </Accordion>
                   )}
 
-                  {!viewRecord && (
+                  {!(isCorrection(declaration) || viewRecord) && (
```

Comment:
We can't this in core as then it changes it for all countries, not only in Farajaland.

The correct place to change this is somewhere in here, in Farajaland repository
https://github.com/opencrvs/opencrvs-farajaland/blob/develop/src/api/application/application-config-default.ts#L48-L51

Reply:
I guess `INFORMANT_SIGNATURE` or `INFORMANT_SIGNATURE_REQUIRED` these configs are deprecated. Beside, if even signature is mandatory, we do not want to show it in correction review page right @rikukissa?